.github/workflows/deploy_mainnet.yml

```diff
@@ -0,0 +1,16 @@
+name: Deploy Mainnet
+
+on:
+  push:
+    tags:
+    - v*
+
+env:
+  GRAPHKEY: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Should we rename the variable to `GRAPHKEY` or etc?

Reply:
not that fast! is still wip, I'm trying it haha